extensions/saba/README.md

```diff
@@ -0,0 +1,21 @@
+<p align="center">
+<img width="60" height="60" alt="Raycast 2023-04-27 at 22 03 33" src="https://user-images.githubusercontent.com/20072792/235139304-0c98f422-2eda-42cb-b5b4-c2af415492ae.png">
+</p>
+
+<h1 align="center">Saba</h1>
+<p align="center">
+Saba is a raycast extension for watching live TV
+</p>
+
+<p align="center">
+<img width="1000" alt="Raycast 2023-04-27 at 22 03 33" src="https://user-images.githubusercontent.com/20072792/235139266-b9358f68-85e2-45c9-afc6-dc75aad6c589.png">
+</p>
+
+### Requirements
+
+- [VLC Media Player](https://www.videolan.org/vlc/) for macOS
```

Comment:
How to get a key to test with?

Reply:
I'm struggling a bit with what saba is, is this legal, or?

I tried the key and get the list, should I be able to do more, seems like the URLs doesn't work